docs/_source/docs/cli.rst

```diff
@@ -17,7 +17,7 @@ following command:
 +----------+------------------------------------------------+
 | shell    | command                                        |
 +==========+================================================+
-| bash     | eval "$(_SCEPTRE_COMPLETE=source sceptre)"     |
+| bash     | eval "$(_SCEPTRE_COMPLETE=source_bash sceptre)"     |
```

Comment:
huh, there's a duplicate description on line 5 & 8.  Would be great if you can fix that as well.

Reply:
Yes of course. Updated.